script/Ciel/Day 7/07_01_CIEL07_2_2.txt

```diff
@@ -645,37 +689,40 @@ It's an enchanting whisper, yet somewhat desolate.
 [sha:ade496559e532232b2861c68e367e3a9c69207d6]{
 -- Page 326, Offset 15596. Mods: @n.
 -- 「予定表。わたしね、子供の頃からリストを作るのが趣味なの。
-"My itinerary. See, one of my hobbies since I was a kid has been making lists.
+"My itinerary. See, ever since I was a kid one of my hobbies has been making lists.
 }
 [sha:098dd3ceadb8c0ed604f8dde215c0c20e2864362]{
 -- Page 326, Offset 15597. Glued.
 -- やりたい事、やらない事、欲しいモノ、要らないモノ。そういうものを順序立ててメモっていくの」
-Want I want to do, what I don't want to do, things I need, things I don't need. I memo down stuff like this――put them in order."
+ What I want to do, what I don't want to do, things I need, things I don't need.
+ I jot down stuff like this to put it in order."
```

Comment:
This is like, "I jot down stuff and put them in order" rather than "jot down to put in order"